# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tar/tar-bzip2.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This adds some historic compatibility for -I and -y to tar. One day we should
obsolete this ...

  - Rene Rebe <rene@exactcode.de>

--- tar-1.15/src/tar.c	2004-12-18 21:01:07.000000000 +0100
+++ tar-1.15-patched/src/tar.c	2004-12-22 08:27:07.469445784 +0100
@@ -431,6 +431,10 @@
    N_("create archive with volume name NAME. At list/extract time, use TEXT as a globbing pattern"), 68 },
   {"bzip2", 'j', 0, 0,
    N_("filter the archive through bzip2"), 68 },
+  {"bzip2", 'I', 0, 0,
+   N_("filter the archive through bzip2"), 68 },
+  {"bzip2", 'y', 0, 0,
+   N_("filter the archive through bzip2"), 68 },
   {"gzip", 'z', 0, 0,
    N_("filter the archive through gzip"), 68 },
   {"gunzip", 0, 0, OPTION_ALIAS, NULL, 68 },
@@ -709,11 +713,9 @@
       break;
       
     case 'I':
-      USAGE_ERROR ((0, 0,
-		    _("Warning: the -I option is not supported;"
-		      " perhaps you meant -j or -T?")));
+      set_use_compress_program_option ("bzip2");
       break;
-      
+
     case 'j':
       set_use_compress_program_option ("bzip2");
       break;
@@ -901,9 +903,7 @@
       break;
       
     case 'y':
-      USAGE_ERROR ((0, 0,
-		    _("Warning: the -y option is not supported;"
-		      " perhaps you meant -j?")));
+      set_use_compress_program_option ("bzip2");
       break;
       
     case 'z':
